features/HomeTab/FlatOffer/Header/Header.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import { View } from "react-native";
+import { IconButton, useTheme, Text } from "react-native-paper";
+
+interface HeaderProp {
+  navigation: any;
+  name: any;
+}
+
+const Header = ({ navigation, name }: HeaderProp) => {
+  const theme = useTheme();
+  const [isLiked, setIsLiked] = useState(false);
+
+  return (
+    <View
+      style={{
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "space-between",
+        marginTop: 10,
+        marginBottom: 10,
+        borderBottomWidth: 1,
+        borderBottomColor: theme.colors.backdrop,
+      }}>
+      <IconButton
+        icon="arrow-left-drop-circle-outline"
```

Comment:
For consistency across whole app, I prefer `"arrow-left-thin-circle-outline"` as arrow back icon. It's not wrong, but in the other component, we use `"arrow-left-thin-circle-outline"` so it would be awesome to stay consistent.